docs/tutorials/HD143006_Part_1.py

```diff
@@ -0,0 +1,211 @@
+
+# opening the fits file
+dfits = fits.open(fname_F)
+# getting the data from the fits file
+clean_fits = dfits[0].data
+header = dfits[0].header
+# closing fits file
+dfits.close()
+
+# the following is to make axis scaling in arcsec (calculating the image extent)
+# get the number of pixels in each direction
+nx = header["NAXIS1"]
+ny = header["NAXIS2"]
+# RA coordinates
+CDELT1 = 3600 * header["CDELT1"]  # Convert from decimal scaling to arcsec
+# DEC coordinates
+CDELT2 = 3600 * header["CDELT2"]  # [arcsec]
+RA = (np.arange(nx) - nx / 2) * CDELT1  # [arcsec]
+DEC = (np.arange(ny) - ny / 2) * CDELT2  # [arcsec]
+# extent needs to include extra half-pixels.
+# RA, DEC are pixel centers
+ext = (
+    RA[0] - CDELT1 / 2,
+    RA[-1] + CDELT1 / 2,
+    DEC[0] - CDELT2 / 2,
+    DEC[-1] + CDELT2 / 2,
+)  # [arcsec]
+
+# plotting the clean fits
+plt.imshow(np.squeeze(clean_fits), origin='lower', extent=ext)
+# limiting the x and y axes so that we only plot the disk
+plt.xlim(left=0.75, right=-0.75)
+plt.ylim(top=0.75, bottom=-0.75)
+# axis labels
+plt.xlabel(r"$\Delta \alpha \cos \delta$ [${}^{\prime\prime}$]")
+plt.ylabel(r"$\Delta \delta$ [${}^{\prime\prime}$]")
+
+# This is the image produced by the CLEAN algorithm used by the DSHARP team.  In Part 1 of this tutorial we will be creating an MPoL Gridder object and  the diagnostic dirty image. [**Part 2**](https://mpol-dev.github.io/MPoL/tutorials/HD143006_Part_2.html) of the tutorial will cover the optimization loop of the model to create the RML image. 
+# 
+# To create the dirty image, we will use the extracted visibilities from the npz file and the MPoL Gridder and Coordinates packages.
+# 
+
+# load extracted visibilities from npz file
+dnpz = np.load(fname_EV) 
+uu = dnpz['uu']
+vv = dnpz['vv']
+weight = dnpz['weight']
+data = dnpz['data']
+
+# Let's quickly plot the U,V visibilities as seen in the [Cross Validation Tutorial](https://mpol-dev.github.io/MPoL/tutorials/crossvalidation.html) and the [Visread docs](https://mpol-dev.github.io/visread/tutorials/introduction_to_casatools.html#Get-the-baselines).
+
+fig, ax = plt.subplots(nrows=1)
+ax.scatter(uu,vv, s=.5, rasterized=True, linewidths=0.0, c='k')
+ax.scatter(-uu,-vv, s=.5, rasterized=True, linewidths=0.0, c='k')
+ax.set_xlabel(r"$u$ [k$\lambda$]")
+ax.set_ylabel(r"$v$ [k$\lambda$]")
+ax.set_title(r'$U$, $V$ Visibilities')
+
+# As you can see, there are very few visibilities > 7,000 ($k\lambda$), and a very dense region of visibilities between -2000 and 2000 ($k\lambda$). This indicates outliers at the higher frequencies while the bulk of our data stems from these lower frequencies.
```

Comment:
I wouldn't call the visibilities at long baselines outliers. The term [outlier](https://www.itl.nist.gov/div898/handbook/prc/section1/prc16.htm) has a specific statistical connotation that usually implies something with the data being suspect. Though the baseline lengths of these visibilities are indeed larger than the rest, the data is still quite valid (and useful). Instead of outlier you could just say there are several groups of long-baseline visibililities as well.